lib/rack/throttle/version.rb

```diff
@@ -1,8 +1,8 @@
 module Rack; module Throttle
   module VERSION
     MAJOR = 0
-    MINOR = 4
```

Comment:
I can bump the version directly on master during deploy. This helps to ensure version changes happen in order. Its more helpful on more active projects, but just good practice.